packages/ubuntu_desktop_installer/pubspec.yaml

```diff
@@ -39,6 +39,8 @@ dependencies:
   tuple: ^2.0.0
   url_launcher: ^6.0.3
   yaru: ^0.0.8
+  form_field_validator: ^1.1.0
```

Comment:
can we keep the dependencies in alphabetical order?

P.S. are you, by any chance, using vscode? if so, check [Pubspec Assist](https://marketplace.visualstudio.com/items?itemName=jeroen-meijer.pubspec-assist) out! it's really nice for managing dependencies, but it places the list of dependencies in alphabetical order (besides flutter sdk dependencies at the top).